static/js/controllers/appcontroller.js

```diff
@@ -56,6 +56,13 @@ define(["jquery", "angular", "underscore"], function($, angular, _) {
 					videoLeakyBucket: true,
 					videoNoiseReduction: false
 				},
+				webrtc: {
+					turn: {
```

Comment:
Make this use a key which is related to the specific feature. Like clientSideTurn or ownTurn.